recipes/mpdecimal/mpdecimal.inc

```diff
@@ -0,0 +1,21 @@
+DESCRIPTION = "Package for correctly-rounded arbitrary precision decimal floating point arithmetic"
+LICENSE = "custom"
```

Comment:
uhm, what :)
"License
The latest source and binary distributions on this page are licensed under the Simplified BSD License.
"